src/ray/raylet/node_manager.cc

```diff
@@ -3353,7 +3355,8 @@ std::unique_ptr<AgentManager> NodeManager::CreateDashboardAgentManager(
   // https://github.com/python/cpython/issues/83086
   int agent_id = 0;
   while (agent_id == 0) {
-    agent_id = rand();
+    unsigned int seed = time(nullptr);
```

Comment:
Not sure this is the best way to get random number, i guess cpplint suggest thread unsafety, so you switch to threadsafe interface, my concern is it hurts performance more or less, let me do some research

Reply:
Ok a second look, it happens in the constructor, so no thread-safety issue; let's `// NOLINT` it :)